src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ProviderHandler.java

```diff
@@ -89,33 +60,31 @@ public static void removeProvider(final IStandardRequestManager manager, final I
      * @throws IllegalArgumentException is thrown when the token is not registered to a provider, or when the data stored in the manager is in conflict.
      */
     @SuppressWarnings(Suppression.UNCHECKED)
-    public static void removeProviderInternal(final IStandardRequestManager manager, final IToken token) throws IllegalArgumentException
+    public static void removeProviderInternal(final IStandardRequestManager manager, final IToken<?> token)
     {
-        final IRequestResolverProvider provider = getProvider(manager, token);
-
-        LogHandler.log("Removing provider: " + provider);
+        LogHandler.log("Removing provider: " + token);
 
         //Get the resolvers that are being removed.
-        final ImmutableCollection<IToken> assignedResolvers = getRegisteredResolvers(manager, token);
-        for (final IToken resolverToken : assignedResolvers)
+        final Collection<IToken<?>> assignedResolvers = getRegisteredResolvers(manager, token);
+        for (final IToken<?> resolverToken : assignedResolvers)
         {
             //Skip if the resolver has no requests assigned.
-            if (!manager.getResolverRequestMap().containsKey(resolverToken) || manager.getResolverRequestMap().get(resolverToken).isEmpty())
+            if (!manager.getRequestResolverRequestAssignmentDataStore().getAssignments().containsKey(resolverToken) || manager.getRequestResolverRequestAssignmentDataStore().getAssignments().get(resolverToken).isEmpty())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 220 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)